itou/templates/utils/widgets/duet_date_picker_widget.html

```diff
@@ -1,6 +1,7 @@
 <duet-date-picker
     identifier="{{ widget.attrs.id }}"
     {% if widget.attrs.required %}required{% endif %}
+    {% if widget.attrs.disabled %}disabled="disabled"{% endif %}
```

Comment:
```suggestion
    {% if widget.attrs.disabled %}disabled{% endif %}
```

Ça suffit car [c'est un attribut booléen](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled).